ui/app/AppLayouts/Profile/views/AppearanceView.qml

```diff
@@ -37,20 +38,32 @@ SettingsContentBase {
         appearanceView.updateFontSize(localAccountSensitiveSettings.fontSize)
     }
 
+    readonly property var priv: QtObject {
+        id: priv
+
+        readonly property real savedDpr: {
+            console.warn("!!! DPR:", appearanceView.Screen.devicePixelRatio)
+            const scaleFactorStr = appearanceView.appearanceStore.readTextFile(uiScaleFilePath)
+            console.warn("!!! SCALE FACTOR STR:", scaleFactorStr)
```

Comment:
Do we still want those logs?

Reply:
I'll remove them when it gets tested

Reply:
![image](https://github.com/status-im/status-desktop/assets/5377645/47dc6bf7-c92c-47cd-ae8e-1c15e157c546)

Reply:
Done